pkg/topology/kademlia/kademlia.go

```diff
@@ -186,6 +190,8 @@ type Kad struct {
 	commonBinPrefixes [][]swarm.Address     // list of address prefixes for each bin
 	connectedPeers    *pslice.PSlice        // a slice of peers sorted and indexed by po, indexes kept in `bins`
 	knownPeers        *pslice.PSlice        // both are po aware slice of addresses
+	connRetryBackoff  map[string]uint64     //
```

Comment:
Since the value is often cast to `time.Duration`, I recommend using `int64` instead of `uint64` to avoid possible overflow.